nservicebus/rabbitmq/configuration-api.md

```diff
@@ -107,4 +107,25 @@ If the routing topologies mentioned above isn't flexible enough then take full c
  1. Define the topology by creating a class implementing `IRoutingTopology`
  1. Register it with the transport calling `.UseRoutingTopology` as shown below
 
-snippet:rabbitmq-config-useroutingtopology
\ No newline at end of file
+snippet:rabbitmq-config-useroutingtopology
+
+## Transactions and delivery guarantees
+
+### Version 6 and above
```

Comment:
You mentioning versions, made me realize that I should have said "versions 4 and above" here since it should be talking about RabbitMQ transport versions here and not core, right?